test/fixtures/3p-ad.html

```diff
@@ -14,20 +14,22 @@
   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
   <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
   <script async src="/dist/amp.js"></script>
-  <!-- use custom url to force same origin -->
   <script async custom-element="amp-ad" src="/dist/v0/amp-ad-0.1.max.js"></script>
+  <!-- use custom url to force same origin -->
   <meta name="amp-3p-iframe-src" content="http://localhost:9876/dist.3p/current/frame.max.html">
 </head>
 <body>
   <div class='fill'></div>
-  <amp-ad width=300 height=250
-        type="_ping_"
-        data-id='0'
-        data-url='https://example.com/a?b=c&d=e'
-        data-valid='true'
-        data-custom-value=123
-        data-other_value="foo">
-  </amp-ad>
+  <amp-lightbox>
```

Comment:
nit: add a note that `<amp-lightbox>` is only used to test context.container. One should not add the lightbox script here